app/src/main/java/org/mozilla/focus/screenshot/ScreenshotViewerActivity.java

```diff
@@ -261,6 +261,8 @@ private void initScreenshotInfo(boolean withShare) {
             final ImageSource imageSource;
             mImageUri = Uri.fromFile(new File(mScreenshot.getImageUri()));
             imageSource = ImageSource.uri(mImageUri);
+            mImgScreenshot.setMinimumScaleType(SubsamplingScaleImageView.SCALE_TYPE_CUSTOM);
```

Comment:
The other inits including a setMinimumScaleType is in
`onCreate`
Let's put these there?